clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -2411,8 +2411,7 @@ public DescribeLogDirsResult describeLogDirs(Collection<Integer> brokers, Descri
                     // Query selected partitions in all log directories
                     return new DescribeLogDirsRequest.Builder(new DescribeLogDirsRequestData().setTopics(null));
                 }
-
-                @SuppressWarnings("deprecation")
+                
```

Comment:
please remove this change (unnecessary Indent)

Reply:
Thax remind,I have remove.